product-service/serverless.yml

```diff
@@ -28,11 +28,11 @@ provider:
   lambdaHashingVersion: 20201221
 
   environment:
-    PG_HOST: task-4.cvwsubzpwjwk.eu-west-1.rds.amazonaws.com
-    PG_PORT: 5432
-    PG_DATABASE: task4
-    PG_USERNAME: postgres
-    PG_PASSWORD: goodwin1
+    PG_HOST: 
```

Comment:
From my point of view, It won't work. You need somehow set all necessary variables in the lambda environment.
One of the applicable approaches is to create `.env` file with the list of key-value pairs and add it to `.gitignore` to avoid publication. 
Read more https://www.serverless.com/framework/docs/environment-variables/